app/templates/teacherhome/liveformresponses.html

```diff
@@ -2,8 +2,6 @@
 {% block content %}
 <div class="container-fluid">
     <div class="row">
```

Comment:
Potential Inmprovement:Fix code style. Making changes e.g minimizing gaps between lines and making the code more structured and ordered makes the code more readable.